### PR TITLE
hs1 driving code

### PR DESCRIPTION
**Pull request by dwaltsch**

### CODE REVIEW

File: src/lego_robot/main.c [hunk 1]
```diff
@@ -13,14 +13,14 @@
 void follow_black_line(int, int);
 void drive(int, int, int);
 void drive_to_start();
+void bring_poms_to_Analysis_Center();
```
Comment (schmaenjael) [inline src/lego_robot/main.c]: Lowercase "analysis center"!

File: src/lego_robot/main.c [hunk 2]
```diff
@@ -81,9 +81,27 @@ void drive_to_start()
...
-    while (drive_distance(125000) && is_frontal_collision())
+    while (drive_distance(12500) && is_frontal_collision())
     {
         follow_black_line(60, 60);
     }
+    bring_poms_to_analysis_center();
 }
+
+void bring_poms_to_analysis_center(){
+    printf("poms zum analyse cetner wuerd ufgrueafa \n");
```
Comment (schmaenjael) [inline src/lego_robot/main.c]: What is meant by this? Is this soley used for debugging? If so, please remove.

File: src/lego_robot/main.c [hunk 3]
```diff
@@ -13,14 +13,14 @@
 void follow_black_line(int, int);
 void drive(int, int, int);
 void drive_to_start();
+void bring_poms_to_Analysis_Center();
 boolean is_frontal_collision();
 boolean drive_distance(int);
+boolean is_over_black_line = false;
```
Comment (schmaenjael) [inline src/lego_robot/main.c]: Is this variable actually getting used? Why is it defined globally and not within scope?

**Review by schmaenjael — CHANGES_REQUESTED** [review]
Take a look at the comments, please!

**Review by dwaltsch — COMMENTED** [review]
good

**Review by dwaltsch — COMMENTED** [review]
.

**Review by schmaenjael — APPROVED** [review]
lgtm, better commit messages in the future!

**Pull request merged**